src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -42,6 +42,47 @@ get_edgedev(const xrt_core::device* device)
   return zynq_device::get_dev();
 }
 
+struct bdf 
+{
+  using result_type = query::pcie_bdf::result_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type)
+  {
+    return std::make_tuple(0,0,0);
+  }
+
+};
+
+struct board_name 
+{
+  using result_type = query::board_name::result_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type)
+  {
+    result_type deviceName("edge");
+    std::ifstream VBNV;
+    VBNV.open("/etc/xocl.txt");
```

Comment:
What is this file?
Strange extension in a Unix world...
Should it be under a `/etc/xilinx` or `/etc/default` or whatever?